src/react/backend/location/Locations.tsx

```diff
@@ -1,50 +1,100 @@
-import type { LocationName, LocationType } from '../../../types/config';
-import React, { useCallback, useMemo, useRef } from 'react';
-import Table, * as T from '../../ui-elements/Table';
+import type { LocationType } from '../../../types/config';
+import React, { useMemo, useState } from 'react';
 import { HelpLocationTargetBucket } from '../../ui-elements/Help';
 import {
   canDeleteLocation,
   canEditLocation,
   IngestionCell,
 } from '../../backend/location/utils';
 import {
-  closeLocationDeleteDialog,
-  deleteLocation,
-  openLocationDeleteDialog,
+  deleteLocation, handleErrorMessage
 } from '../../actions';
 import { useDispatch, useSelector } from 'react-redux';
-import { useFilters, useSortBy, useTable } from 'react-table';
 import type { AppState } from '../../../types/state';
 import DeleteConfirmation from '../../ui-elements/DeleteConfirmation';
 import { Warning } from '../../ui-elements/Warning';
 import { push } from 'connected-react-router';
 import { storageOptions } from '../../backend/location/LocationDetails';
 import styled from 'styled-components';
-import { useHeight } from '../../utils/hooks';
 import { XDM_FEATURE } from '../../../js/config';
-const initialSortBy = [
-  {
-    id: 'name',
-    desc: false,
-  },
-];
-const CustomBody = styled(T.Body)`
-  flex: 1;
-  overflow: visible;
-`;
-export const Sizer = styled.div`
-  height: ${(props) => (props.height ? `${props.height}px` : '300px')};
-  width: 100%;
-`;
-const Container = styled.div`
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-  min-width: 500px;
+import { TitleRow as TableHeader } from '../../ui-elements/TableKeyValue';
+import { Table, Button } from '@scality/core-ui/dist/next';
+import { useHistory } from 'react-router-dom';
+import { spacing } from '@scality/core-ui/dist/style/theme';
+import { useMutation } from 'react-query';
+
+const InlineButton = styled(Button)`
+  height: ${spacing.sp24};
+  margin-left: ${spacing.sp16};
 `;
 
+const renderActionButtons = (rowValues: any) => {
```

Comment:
You should try not use `any` everywhere but use real typing instead else it defeat the purpose of using typescript :/